backend/app_logging/example.py

```
@@ -0,0 +1,267 @@
+
+from flask import Flask, request, jsonify
+from flask_jwt_extended import JWTManager, jwt_required, create_access_token
+
+# Import our logging components
+from app_logging.config import setup_logging, get_logger
+from app_logging.middleware import setup_request_logging
+from app_logging.decorators import log_route, log_function, log_database_operation, log_performance
+
+# Setup Flask app
+app = Flask(__name__)
+app.config['JWT_SECRET_KEY'] = 'demo-secret-key'
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "JWT_SECRET_KEY"

(S105)